release-notes/aws/v9.0.1.md

```diff
@@ -0,0 +1,34 @@
+## :zap: Giant Swarm Release 9.0.1 for AWS is now active for you! :zap:
+
+TODO
```

Comment:
@cokiengchiara I've added the changelogs.

I can't find the questions you had for releases. But if you want to post them I'm happy to answer as that might help with why we're doing this.